nikola/conf.py.in

```diff
@@ -25,7 +25,7 @@ SITE_URL = ${SITE_URL}
 # This is the URL where Nikola's output will be deployed.
 # If not set, defaults to SITE_URL
 # BASE_URL = ${SITE_URL}
-BLOG_EMAIL = ${BLOG_EMAIL}
+BLOG_EMAIL = ${BLOG_EMAIL}  # (translatable)
```

Comment:
I don’t think `BLOG_EMAIL` needs to be translatable. You can just make it a regular string (as `blog_url` is).